docs/docs/05-migrations/13-v8-to-v9.md

```diff
@@ -47,6 +47,21 @@ Please use the new functions `path.Setup`, `path.SetupClients`, `path.SetupConne
 - Functions `ConstructUpdateTMClientHeader` and `ConstructUpdateTMClientHeaderWithTrustedHeight` of `TestChain` type have been replaced with `IBCClientHeader`. This function will construct a `07-tendermint` header to update the light client on the counterparty chain. The trusted height must be passed in as a non-zero height.
 - `GetValsAtHeight` has been renamed to `GetTrustedValidators`
 
+### ICS27 - Interchain Accounts
+
+In [#5785](https://github.com/cosmos/ibc-go/pull/5785) the list of arguments of the `NewKeeper` constructor function of the host submodule was extended with an extra argument for the gRPC query router that the submodule uses when executing a [`MsgModuleQuerySafe`](https://github.com/cosmos/ibc-go/blob/eecfa5c09a4c38a5c9f2cc2a322d2286f45911da/proto/ibc/applications/interchain_accounts/host/v1/tx.proto#L41-L51) to perform queries that are module safe:
+
+```diff
+func NewKeeper(
+  cdc codec.Codec, key storetypes.StoreKey, legacySubspace icatypes.ParamSubspace,
+  ics4Wrapper porttypes.ICS4Wrapper, channelKeeper icatypes.ChannelKeeper,
+  portKeeper icatypes.PortKeeper,	accountKeeper icatypes.AccountKeeper, 
+  scopedKeeper exported.ScopedKeeper, msgRouter icatypes.MessageRouter,
++ queryRouter icatypes.QueryRouter, 
+  authority string,
```

Comment:
```suggestion
- authority string, 
+ queryRouter icatypes.QueryRouter, authority string,
```
SUPER nit: I think the styling looks better

Reply:
I put it like to this to highlight that the only addition was the query router, but can change to what you propose, if you prefer.

Reply:
They are both equivalent @DimitrisJim. And I'm fine either way @crodriguezvega